openupgrade_scripts/scripts/digest/14.0.1.1/post-migration.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (C) 2021 Open Source Integrators
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(env.cr, "digest", "14.0.1.1/noupdate_changes.xml")
```

Comment:
Seeing the noupdate data, I don't think we should change the ones customized by the customer.

Reply:
ok, I commented the code in the noupdate_changes.xml file